resources/views/helper.php

```diff
@@ -12,7 +12,7 @@
 
 /**
  * A helper file for Laravel, to provide autocomplete information to your IDE
- * Generated for Laravel <?= $version ?> on <?= date("Y-m-d H:i:s") ?>.
+ * Generated for Laravel <?= $version ?><?php if(!(isset($hide_datetime) && $hide_datetime)): ?> on <?= date("Y-m-d H:i:s") ?><?php endif;?>.
```

Comment:
The PR always set the variable for the templates, so it seems to me `isset` isn't necessary?

Reply:
Might I suggest a different approach?

In stead of "if not hide: echo $date; endif" how about the inverse / positive check:
- call the var `show_datetime`
- default to `true`
- write a "positive" check

```suggestion
 * Generated for Laravel <?= $version ?><?php if ($show_datetime) ?> on <?= date("Y-m-d H:i:s") ?><?php endif;?>.
```

Seems less cluttered and also more straight to the point.

(applies also for the check in the other file)